src/components/Header.js

```diff
@@ -24,12 +36,20 @@ const HeaderStyled = styled.section`
 
 const AppName = styled.h1`
   z-index: 1;
-  font-size: 19px;
+  font-size: 18px;
   padding: 12px;
   user-select: none;
 `
 
 const Logo = styled.img`
   position: absolute;
-  width: 42px;
+  width: 38px;
+`
+
+const Help = styled.img`
+  color: goldenrod;
+  position: Absolute;
```

Comment:
Maybe change to absolute with lower case a, for the sake of general css conventions :)